docs: Physical machine auth (#215)
chaos-mesh/website: docs/simulate-physical-machine-chaos.md

## Deploy Chaosd server

> Before creating PhysicalMachineChaos experiments using Chaos Mesh, you need to deploy Chaosd in service mode to all physical or virtual machines that are going to be injected with faults. After deploying Chaosd, run Chaosd Server as follows:

Author: done

> 1. Deploy Chaosd: for the deployment method of Chaosd, refer to [Download and deploy Chaosd](chaosd-overview.md#download-and-deploy).

Author: done

>    :::note

Author: done

>    When using Chaos Mesh v2.1.0, you need to deploy Chaosd [v1.1.0](https://github.com/chaos-mesh/chaosd/releases/tag/v1.1.0).

Author: done

   :::

2. Generate TLS certificate, create `PhysicalMachine`: After deploying Chaosd, you need to generate TLS certificate and create a `PhysicalMachine` within the Kubernetes cluster **before** starting Chaosd Server. For more information on how to generate TLS certificates, please refer to [Generate TLS certs for Choasd](chaosctl-tool.md#generate-tls-certs-for-choasd).

> 3. Start Chaosd Server:
> 
>    + After the deployment is complete, run the following command to start Chaosd in service mode:
> 
>       ```bash
>       chaosd server --https-port 31768 --CA=/etc/chaosd/pki/ca.crt --cert=/etc/chaosd/pki/chaosd.crt --key=/etc/choasd/pki/chaosd.key
>       ```
> 
>       :::note
> 
>       The paths of certificates are the default output paths of `Chaosctl`. If you specify another path manually when generating certificates, please replace the corresponding file path.
> 
>       :::
> 
> 
>    + If no TLS certificate is configured, use the following command to start Chaosd in service mode. This is **not recommended** considering cluster security:
> 
>       ```bash
>       chaosd server --port 31767
>       ```
> 
> ## Create experiments using Chaos Dashboard

Reviewer: ```suggestion
2. Start Chaosd server:

- After generating the TLS certificate through Chaosctl, run the following command to start Chaosd in service mode:

    ```bash
    chaosd server --https-port 31768 --CA=/etc/chaosd/pki/ca.crt --cert=/etc/chaosd/pki/chaosd.crt --key=/etc/choasd/pki/chaosd.key
    ```

    :::note

    The TSL certificates are saved to the defaulted output directory of Chaosctl. If you manually specified another directory when generating certificates, replace the directory in the command line with the corresponding one.

    :::

- If the TLS certificate is not generated through Chaosctl, you can run the following command to start Chaosd in service mode. However, for the security of your clusters, this is **not** recommended.

    ```bash
    chaosd server --port 31767
    ```

## Create experiments using Chaos Dashboard
```

Author: done

| Configuration item | Type | Description	 | Default value | Required | Example |
| :-- | :-- | :-- | :-- | :-- | :-- |

> | `action` | string | Defines the actions of physical machines faults, optional values are as follows: "stress-cpu", "stress-mem", "disk-read-payload", "disk-write-payload", "disk-fill", "network-corrupt", "network-duplicate", "network-loss", "network-delay", "network-partition", "network-dns", "process", "jvm-exception", "jvm-gc", "jvm-latency", "jvm-return", "jvm-stress", "jvm-rule-data", "clock" | None | Yes | "stress-cpu" |
> | `address` | string array | Selects the `address` of Chaosd service to inject faults, only one of `address` or `selector` could be specified | [] | Yes | ["192.168.0.10:31767"] |
> | `selector` | struct | Specifies the target PhysicalMachine. For details, refer to [Define the experiment scope](define-chaos-experiment-scope.md), only one of `address` or `selector` could be specified | None | No | |
> | `mode` | string | Specifies the mode of the experiment. The mode options include `one` (selecting a random PhysicalMachine), `all` (selecting all eligible PhysicalMachines), `fixed` (selecting a specified number of eligible PhysicalMachines), `fixed-percent` (selecting a specified percentage of PhysicalMachines from the eligible PhysicalMachines), and `random-max-percent` (selecting the maximum percentage of PhysicalMachines from the eligible PhysicalMachines). | None | Yes | `one` |

Reviewer: ```suggestion
| `action` | string | Defines the actions of physical machines faults, optional values are as follows: `"stress-cpu"`, `"stress-mem"`, `"disk-read-payload"`, `"disk-write-payload"`, `"disk-fill"`, `"network-corrupt"`, `"network-duplicate"`, `"network-loss"`, `"network-delay"`, `"network-partition"`, `"network-dns"`, `"process"`, `"jvm-exception"`, `"jvm-gc"`, `"jvm-latency"`, `"jvm-return"`, `"jvm-stress"`, `"jvm-rule-data"`, `"clock`" | None | Yes | `"stress-cpu"` |
| `address` | string array | Selects the `address` of Chaosd service to inject faults, only one of `address` or `selector` could be specified | [] | Yes | ["192.168.0.10:31767"] |
| `selector` | struct | Specifies the target PhysicalMachine. For details, refer to [Define the experiment scope](define-chaos-experiment-scope.md), only one of `address` or `selector` could be specified | None | No | |
| `mode` | string | Specifies the mode of the experiment. The mode options include `one` (selecting a random PhysicalMachine), `all` (selecting all eligible PhysicalMachines), `fixed` (selecting a specified number of eligible PhysicalMachines), `fixed-percent` (selecting a specified percentage of PhysicalMachines from the eligible PhysicalMachines), and `random-max-percent` (selecting the maximum percentage of PhysicalMachines from the eligible PhysicalMachines). | None | Yes | `one` |
```

Author: done

| `value` | string | Provides a parameter for the `mode` configuration, depending on `mode`. For example, when `mode` is set to `fixed-percent`, `value` specifies the percentage of PhysicalMachines. | None | No | `1` |

> | `duration` | string | Specifies the duration of experiments | None | Yes | 30s |

Reviewer: ```suggestion
| `duration` | string | Specifies the duration of experiments | None | Yes | `30s` |
```

Author: done
